src/TibiaHousesOverviewV3.go

```diff
@@ -168,3 +98,76 @@ func TibiaHousesOverviewV3(c *gin.Context) {
 	// return jsonData
 	TibiaDataAPIHandleSuccessResponse(c, "TibiaHousesOverviewV3", jsonData)
 }
+
+func houseFetcher(baseURL, houseType string, done chan struct{}, outputChan chan House) {
+	// Getting data with TibiadataHTMLDataCollectorV3
+	BoxContentHTML := TibiadataHTMLDataCollectorV3(baseURL + TibiadataQueryEscapeStringV3(houseType))
+
+	// Loading HTML data into ReaderHTML for goquery with NewReader
+	ReaderHTML, err := goquery.NewDocumentFromReader(strings.NewReader(BoxContentHTML))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	ReaderHTML.Find(".BoxContent table tr").Each(func(index int, s *goquery.Selection) {
+		// Storing HTML into HousesDivHTML
+		HousesDivHTML, err := s.Html()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		// Removing linebreaks from HTML
+		HousesDivHTML = TibiadataHTMLRemoveLinebreaksV3(HousesDivHTML)
+
+		// Regex to get data for record values
+		regex1 := regexp.MustCompile(`<td.*><nobr>(.*)<\/nobr><\/td><td.*><nobr>([0-9]+).sqm<\/nobr><\/td><td.*><nobr>([0-9]+)(k+).gold<\/nobr><\/td><td.*><nobr>(.*)<\/nobr><\/td>.*houseid" value="([0-9]+)"\/><div.*`)
+		subma1 := regex1.FindAllStringSubmatch(HousesDivHTML, -1)
+
+		if len(subma1) > 0 {
+			// Default vars
+			var (
+				IsRented, IsAuctioned bool
+				AuctionBid            int
+				AuctionLeft           string
+			)
+
+			s := subma1[0][5]
+			switch {
+			case strings.Contains(s, "rented"):
+				IsRented = true
+			case strings.Contains(s, "no bid yet"):
+				// nothing to set?
+			case strings.Contains(s, "auctioned"):
```

Comment:
```suggestion
			case strings.Contains(s, "auctioned"):
```

even if there is no bid, it's still being auctioned.. so this should be removed